check_license_go_code.sh

```diff
@@ -16,16 +14,29 @@ cat > license.tmp <<EOF
 //    limitations under the License.
 EOF
 lines=$(cat license.tmp | wc -l)
+# we need to add two extra lines missing from the license preamble
+# // Copyright <copyrigt_year> Northern.tech AS
+# //
+lines=$(($lines + 2))
 
 ret=0
 for each in $(find . -type f \( ! -regex '.*/\..*' ! -path "./Godeps/*" ! -path "./vendor/*" -name '*.go' \)); do
-  echo "Checking $each for correct license header"
-  head -n $lines $each | diff -qu license.tmp - > /dev/null
+  modified_year=$(date +%Y -r $each)
```

Comment:
You cannot check it like this, because git always checks out all files at today's date. Try this instead:

```
git log -n1 --format=%ad --date=format:%Y -- $each
```

Reply:
Thanks for spotting this. Fixed!